### PR TITLE
Attempt to count TCZ trials

### PR DESCRIPTION
**Pull request by rando2**
### Description of the proposed additions or changes

@agitter I was hoping to tabulate the number of tocilizumab trials (and then potentially do this for some of the other therapeutics we highlight in the Therapeutics section as well). I would love to know if you think this is a reasonable approach to exporting the count from the EBM data -- and apologies if I've totally missed the mark! If this is reasonable, I was hoping to add in a reference to the tocilizumab_ct variable in the text. Thank you so much for your help in getting acquainted with CI!

### Suggested reviewers (optional)

@agitter or @rdvelazquez 

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: ebmdatalab/generate-ebmdatalab-stats.py [hunk 1]
```diff
@@ -123,6 +123,9 @@ def main(args):
     ebm_stats['ebm_trials_figure'] = \
         f'https://github.com/greenelab/covid19-review/raw/$FIGURE_COMMIT_SHA/{args.output_figure}.svg'
     
+    # Tabulate number of trials for pharmaceuticals of interest
+    ebm_stats['tocilizumab_ct'] = trials_df['intervention'].str.contains('tocilizumab', case=False).sum()
```
Comment (agitter) [inline ebmdatalab/generate-ebmdatalab-stats.py:127]:
```suggestion
    ebm_stats['ebm_tocilizumab_ct'] = trials_df['intervention'].str.contains('tocilizumab', case=False).sum()
```

Your approach is correct, and this should work.  I'm using the convention where the variable prefix (`ebm_`) denotes the data source to help with later debugging if needed.

**Comment by rando2** [comment]
@agitter I'm getting 68 this morning, which sounds pretty close! And that is a really important note, thank you so much!

**Comment by agitter** [comment]
@rando2 your update worked.  The last nightly build added `"ebm_tocilizumab_ct": "68",` to `ebmdatalab-stats.json`

**Comment by rando2** [comment]
Yay! Thank you so much for checking and for the help, @agitter!

**Pull request merged**